nextflow-genomics-cluster-ubuntu/azuredeploy.customimage.parameters.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "nextflowDockerAdmin"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "dnsNameForJumpBox": {
+      "value": "nextflowdockerdns"
+    },
+    "vmImageReference": {
```

Comment:
See the contribution guide for how to use images in samples

Reply:
There isn't a dependency on the custom image, was just a sample for how to use one if a user wanted too. Hope that's ok.